os/mm/kmm_heap/kmm_realloc.c

```diff
@@ -99,13 +99,13 @@ void *kmm_realloc_at(int heap_index, void *oldmem, size_t size)
 	ARCH_GET_RET_ADDRESS
```

Comment:
Please add checking size for kmm_realloc_at also, `if (size == 0)`.

Reply:
added